db/dcrpg/internal/stakestmts.go

```diff
@@ -0,0 +1,218 @@
+package internal
+
+const (
+	// Tickets
+
+	CreateTicketsTable = `CREATE TABLE IF NOT EXISTS tickets (  
+		id SERIAL PRIMARY KEY,
+		tx_hash TEXT NOT NULL,
+		block_hash TEXT NOT NULL,
+		block_height INT4,
+		purchase_tx_db_id INT8,
+		stakesubmission_address TEXT,
+		is_multisig BOOLEAN,
+		is_split BOOLEAN,
+		num_inputs INT2,
+		price FLOAT8,
+		fee FLOAT8,
+		spend_type INT2,
+		spend_height INT4,
+		spend_tx_db_id INT8
+	);`
+
+	// Insert
+	insertTicketRow0 = `INSERT INTO tickets (
+		tx_hash, block_hash, block_height, purchase_tx_db_id,
+		stakesubmission_address, is_multisig, is_split,
+		num_inputs, price, fee, spend_type)
+	VALUES (
+		$1, $2, $3,	$4,
+		$5, $6, $7,
+		$8, $9, $10, $11) `
+	insertTicketRow = insertTicketRow0 + `RETURNING id;`
+	// insertTicketRowChecked = insertTicketRow0 + `ON CONFLICT (tx_hash, block_hash) DO NOTHING RETURNING id;`
+	upsertTicketRow = insertTicketRow0 + `ON CONFLICT (tx_hash, block_hash) DO UPDATE 
```

Comment:
should also be a do nothing, but not for the reasons mentioned in the other comment.